src/modules/core/components/Numeral/Numeral.tsx

```diff
@@ -54,19 +53,15 @@ const Numeral = ({
   reducedOutput = true,
   ...props
 }: Props) => {
-  const convertedNum =
-    typeof unit === 'string'
-      ? formatUnits(value, unit)
-      : moveDecimal(value.toString(10), -(unit || 0));
-
-  const largeNumberTruncate = convertedNum > 10 ** 4 ? 0 : truncate;
-
-  const formattedNumber = formatNumber({
+  const formattedNumber = formatTokenValue({
+    unit,
+    value,
     prefix,
     suffix,
     integerSeparator,
-    truncate: reducedOutput ? largeNumberTruncate : truncate,
-  })(parseFloat(convertedNum));
+    truncate,
+    reducedOutput,
+  });
```

Comment:
Can you explain this change please?

Also this needs very rigorous testing, as we use `Numeral` in a hell of alot of places. I just want to make sure we didn't borked some other place of the dapp

Reply:
For the total tokens in  the Figma specs token amount & token symbol are of different font-sizes. I have no way to separate them with using Numeral component as it uses a span element.
So I created a helper that just extracts the logic for the amount & symbol from the Numeral. It's the same logic, just encapsulated in a helper. I then use this helper in my `TokensTab` component & can split the amount & symbol & style them differently.

I've already checked if it's working fine but can do some more testing.

<img width="459" alt="Screenshot 2021-03-02 at 11 37 10" src="https://user-images.githubusercontent.com/34057551/109643068-a66f2e80-7b4b-11eb-9205-fefb38112cd6.png">